src/App.js

```diff
@@ -14,7 +14,6 @@ import Register from './views/Register';
 import TempHome from './components/TempHome';
 
 
-
```

Comment:
Remove this new blank line. Let's have only two empty lines between import and class declaration.